app/controllers/users_controller.rb

```diff
@@ -1,6 +1,6 @@
 class UsersController < ApplicationController
-  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
-  before_action :load_user, only: [:show, :edit, :update, :destroy]
+  before_action :logged_in_user, only: %i(index edit update destroy)
+  before_action :load_user, only: %i(show edit update following followers)
```

Comment:
đổi thành bên dưới cho gọn code 
```suggestion
  before_action :load_user, except: %i(new index create)
```